roda-ui/roda-wui/src/main/resources/config/theme/Welcome.html

```
@@ -1,352 +1,129 @@
-                                    <div class="second-column">
-                                        <div class="highlight-image">
-                                            <img src="api/v2/themes?resource-id=images/vendor-independent.svg"/>
-                                        </div>
-                                    </div>
-                                </div>
-                            </div>
-                        </div>
-                    </div>
-
-                </div>
+    <div class="eterna-welcome__search" role="search">
```

HTMLHint (1.9.2)

[error] Tag must be paired, missing: [ </div> ], start tag match failed [ <div class="eterna-welcome__search" role="search"> ] on line 6.

(tag-pair)